assets/js/steps/check-in-items.js

```diff
@@ -33,6 +34,9 @@ import { faBook } from '@fortawesome/free-solid-svg-icons';
 function CheckInItems({ actionHandler }) {
     const context = useContext(MachineStateContext);
     const [scannedBarcode, setScannedBarcode] = useState('');
+    const [activerBanner, setActiverBanner] = useState(false);
```

Comment:
"activerBanner" => "activeBanner"
throughout the file.

Reply:
upsi